packages/flutter/test/material/ink_well_test.dart

```diff
@@ -2214,4 +2214,28 @@ testWidgetsWithLeakTracking('InkResponse radius can be updated', (WidgetTester t
 
     await gesture.up();
   });
+
+  testWidgetsWithLeakTracking('try out hoverDuration property', (WidgetTester tester) async {
+    final List<String> log = <String>[];
+
+    await tester.pumpWidget(Directionality(
+      textDirection: TextDirection.ltr,
+      child: Material(
+        child: Center(
+          child: InkWell(
+            hoverDuration: const Duration(milliseconds: 1000),
+            onTap: () {
+              log.add('tap');
+            },
+          ),
+        ),
+      ),
+    ));
+
+    await tester.tap(find.byType(InkWell), pointer: 1);
+    await tester.pump(const Duration(seconds: 1));
+
+    expect(log, equals(<String>['tap']));
```

Comment:
Could you explain how this test works?  Would it actually fail without the change in this PR?  I expected it to test a color or something.

Reply:
The test would fail without this pr because it's making use of the new `hoverDuration` property.